src/Microsoft.Identity.Client/Features/ConfidentialClient/ConfidentialClientApplication.cs

```diff
@@ -389,6 +389,26 @@ async Task<AuthenticationResult> IConfidentialClientApplicationWithCertificate.A
             return await handler.CreateAuthorizationUriAsync().ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// This method should be used when you have a solution using ADAL 2.x and 
+        /// caching a refresh token, and you want to migrate to MSAL.NET.
+        /// During the migration process, it enables you to store in MSAL.NET token
+        ///  cache an access token and refresh token corresponding to <paramref="refreshToken"/>.
+        /// From there you will be able to use MSAL.NET new API, in particular
+        /// AcquireTokenSilentAsync() which will renew the user token.  
+        /// </summary>
```

Comment:
Jean-Marc should sign-off on the text and we should ensure to add an aka.ms link to talk more about the usages of this. This is not only for migration, but also for integration scenarios.

Reply:
after talking with JM, I updated the comment to be more precise.

Reply:
Looking at creating blog post atm

Reply:
Post created
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/Adal-to-Msal#adalnet-v2x-migration-to-msalnet-2x